### ISSUE TITLE
Cross Platform (Mac vs PC) Fails Opening Solution

### ISSUE THREAD

**Issue by adamhill** [Unity-Technologies/BoatAttack#92]
In Unity 2019.3.11f1, same in 2019.3.12f1

On Windows 10 (1903) - Project opens up fine
On OSX (Catalina 10.15.4) - Project opens with the error:

```
MissingReferenceException: The variable defaultFoamRamp of WaterResources doesn't exist anymore.
You probably need to reassign the defaultFoamRamp variable of the 'WaterResources' script in the inspector.
UnityEngine.Texture2D.GetPixelBilinear (System.Single u, System.Single v) (at /Users/builduser/buildslave/unity/build/Runtime/Export/Graphics/Texture.cs:599)
WaterSystem.Water.GenerateColorRamp () (at Packages/com.verasl.water-system/Scripts/Water.cs:321)
WaterSystem.Water.Init () (at Packages/com.verasl.water-system/Scripts/Water.cs:153)
WaterSystem.Water.OnEnable () (at Packages/com.verasl.water-system/Scripts/Water.cs:73)
```

In the inspector, I do not see a 'defaultFoamRamp' to assign to
![image](https://user-images.githubusercontent.com/188638/80630880-8433ee00-8a1a-11ea-9c09-cf5313441465.png)

But in the source it is there
![image](https://user-images.githubusercontent.com/188638/80631021-bfceb800-8a1a-11ea-9e87-828a1b92d075.png)

**Comment by adamhill** [comment]
Magically fixed itself in 2019.3.13f. closing.

**Comment by BeshkekArt** [comment]
AHAHAH HOOWW
